relayer/src/event/monitor.rs

```diff
@@ -336,32 +354,61 @@ impl EventMonitor {
 
             let result = rt.block_on(async {
                 tokio::select! {
-                    Some(batch) = batches.next() => Ok(batch),
+                    Some(batch) = batches.next() => batch,
                     Some(err) = self.rx_err.recv() => Err(Error::WebSocketDriver(err)),
                 }
             });
 
             match result {
                 Ok(batch) => self.process_batch(batch).unwrap_or_else(|e| {
-                    warn!(chain.id = %self.chain_id, "{}", e);
+                    error!(chain.id = %self.chain_id, "{}", e);
                 }),
+                Err(Error::SubscriptionCancelled(reason)) => {
+                    error!(chain.id = %self.chain_id, "subscription cancelled, reason: {}", reason);
```

Comment:
I see that the error messages is rendered as follows

> Jul 21 15:25:29.063 ERROR subscription cancelled, reason: Server error: subscription was cancelled (reason: client is not pulling messages fast enough) (code: -32000) chain.id=ibc-0


It may improve log readability if we put the chain id as a tag, just after the `ERROR` message, rendering the line similar to other log output, for instance:

> Jul 21 15:25:29.063 ERROR [ibc-0] subscription cancelled, reason: Server error: subscription was cancelled (reason: client is not pulling messages fast enough) (code: -32000)